File: src/main/java/com/examples/testing/withfeaturecheck/incorrect/java/usesloop/Question.java
package com.examples.testing.withfeaturecheck.incorrect.java.usesloop;

import edu.illinois.cs.cs125.questioner.lib.Incorrect;

@Incorrect(reason = "features")
public class Question {
  int sumTo(int range) {
    int sum = 0;
    for (int i = 0; i <= range; i++) {
      sum += i;
    }
    return sum;
  }
}

File: src/main/java/com/examples/testing/withfeaturecheck/incorrect/kotlin/usesloop/Question.kt
package com.examples.testing.withfeaturecheck.incorrect.kotlin.usesloop

import edu.illinois.cs.cs125.questioner.lib.Incorrect

@Incorrect(reason = "features")
@Suppress("ForEachOnRange")
class Question {
  fun sumTo(range: Int): Int {
    var sum = 0
    (0..range).forEach { i ->
      sum += i
    }
    return sum
  }
}